sessionctx/variable/sysvar.go

```diff
@@ -496,12 +496,6 @@ var defaultSysVars = []*SysVar{
 		SetMaxDeltaSchemaCount(TidbOptInt64(val, DefTiDBMaxDeltaSchemaCount))
 		return nil
 	}},
-	{Scope: ScopeGlobal, Name: TiDBEnableChangeMultiSchema, Value: BoolToOnOff(DefTiDBChangeMultiSchema), Hidden: true, Type: TypeBool, SetGlobal: func(s *SessionVars, val string) error {
-		EnableChangeMultiSchema.Store(TiDBOptOn(val))
-		return nil
-	}, GetGlobal: func(s *SessionVars) (string, error) {
-		return BoolToOnOff(EnableChangeMultiSchema.Load()), nil
-	}},
```

Comment:
Can you add it in here: https://github.com/pingcap/tidb/blob/81cf12ebf651fa60d2bd1938ad22919faf3ba65e/sessionctx/variable/removed.go#L35-L51

Hopefully scripts did not depend on this variable being present - but since there is a graceful way to remove a variable we might as well use it.